### PR TITLE
adding info for dot_net_core

### PR DESCRIPTION
**Pull request by btardif**
fyi @bradygaster

### CODE REVIEW

File: Runtime_Support/dot_net_core.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# .NET Core on App Service
+
+## .NET Core Update Policy
+
+App Service upgrades the underlying .NET Core runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
+
+### End of Life
+
+Once a version of .NET Core has reached it's end of life (EOL) it will no longer be available from Runtime Stack selection dropdown.
```
Comment (timheuer) [inline Runtime_Support/dot_net_core.md]: Perhaps put the support policy link here around the ```.NET Core has reached it's end of life (EOL)```

Comment (btardif) [reply Runtime_Support/dot_net_core.md]: @timheuer i'm trying to stay consistent with what i'm doing for other run-time stacks:

- [PHP](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/php_support.md)
- [Python](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/python_support.md)

File: Runtime_Support/dot_net_core.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+|---------------| -------------- | ----------------- |---------------- |
+| .NET Core 1.0 | End of Life    | June 27 2019      | Windows & Linux |
+| .NET Core 1.1 | End of Life    | June 27 2019      | Windows & Linux |
+| .NET Core 2.0 | End of Life    | October 1, 2018   | Windows & Linux |
+| .NET Core 2.1 | LTS            | August 21, 2021   | Windows & Linux |
+| .NET Core 2.2 | Maintenance    | December 23, 2019 | Windows & Linux |
+| .NET Core 3.0 | Maintenance    | March 3, 2020     | Windows & Linux |
+| .NET Core 3.1 | LTS            | n/a               | Windows & Linux |
+
+[.NET Core Support timeline](https://dotnet.microsoft.com/platform/support/policy/dotnet-core)
```
Comment (btardif) [inline Runtime_Support/dot_net_core.md]: @timheuer I'm trying to keep the structure of this docs consistent:

## Update Policy
## EOL mechanics
## Suport Timeline
### Link to official support docs

Comment (timheuer) [reply Runtime_Support/dot_net_core.md]: Fair...consistency is good...didn't see the other docs.

**Pull request merged**